cmd/server.go

```diff
@@ -519,7 +520,11 @@ var boolFlags = map[string]boolFlag{
 		description: "Disable atlantis global apply lock in UI",
 	},
 	DiscardApprovalOnPlanFlag: {
-		description:  "Enables the discarding of approval if a new plan has been executed. Currently only Github is supported",
+		description:  "Enables the discarding of approval if a new plan has been executed. Currently supported on GitHub, GitLab, and Gitea",
```

Comment:
While writing this PR, I noticed that Gitlab and Gitea supported the DiscardApprovals method, so I added them to the description and documentation.